config/helpers.php

```diff
@@ -413,46 +414,38 @@ function option(string $key, $default = null)
 
 if (Helpers::hasOverride('page') === false) { // @codeCoverageIgnore
     /**
-     * Fetches a single page or multiple pages by
-     * id or the current page when no id is specified
+     * Fetches a single page by id or
+     * the current page when no id is specified
      *
-     * @param string|array ...$id
-     * @return \Kirby\Cms\Page|\Kirby\Cms\Pages|null
-     * @todo reduce to one parameter in 3.7.0 (also change return and return type)
+     * @param string|null $id
+     * @return \Kirby\Cms\Page|null
      */
-    function page(...$id)
+    function page($id = null)
```

Comment:
Since `$site->find()` also only accepts strings:

```suggestion
    function page(?string $id = null)
```